src/Api/Vault/Controllers/SecurityTaskController.cs

```diff
@@ -80,6 +83,18 @@ public async Task<ListResponseModel<SecurityTasksResponseModel>> ListForOrganiza
         return new ListResponseModel<SecurityTasksResponseModel>(response);
     }
 
+    /// <summary>
+    /// Retrieves security task metrics for an organization.
+    /// </summary>
+    /// <param name="orgId">The organization Id</param>
+    [HttpGet("{orgId:guid}/metrics")]
+    public async Task<SecurityTaskMetricsResponseModel> GetTaskMetricsForOrg([FromRoute] Guid orgId)
```

Comment:
🎨 Shorter naming is used for the organization which is outside the norm -- make this look like the other methods.